backward compability

geojak opened the pull request:
old default was 2, new default should be unchanged to prevent peoples framerate to change without them nowing becuase they have still some settings from the old version stored.

geojak committed:
backward compability

old default was 2, new default should be unchanged to prevent peoples framerate to change without them nowing becuase they have still some settings from the old version stored.

Mtarnuhal commented:
Can you put patch-3 and patch 4 together since they are the same.
Also a good thing to do is increase the version if we patch something.
V Major.Minor.Patch

geojak commented:
You are right, I will fix it later today. I got a interesting discussion on discord I wanted to tell you about. So this guy has a problem with the blue line in the bottom left  saying HC completion %, constantly appearing and disappearing. Turned out now that his cookies to next HC is negative. He will sent me his save file later. 

Anyway I wanted to propose to delete that HC% line.  I sent you some screenshots if you are interested
From: Mtarnuhal
Sent: 30 October 2018 13:27
To: Mtarnuhal/FrozenCookies
Cc: geojak; Author
Subject: Re: [Mtarnuhal/FrozenCookies] backward compability (#9)

Can you put patch-3 and patch 4 together since they are the same.
Also a good thing to do is increase the version if we patch something.
V Major.Minor.Patch
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

Mtarnuhal committed:
Correct operator

Mtarnuhal committed:
FC_button position fix

geojak committed:
added patch 4 into patch 3

backward compability

geojak committed:
redundant info 

deleted the blue HC % line in the botton left because the vanilla bar at the top right is okay enough for this on new games (i guess) and useless very soon for everybody

commented out some HC information in the frozen cookie tab that seems not too usefull. (time till next HC might be usefull in the first 1-2 ancensions)

geojak committed:
Merge pull request #1 from geojak/geojak-patch-4

redundant info

geojak commented:
i deleted/commented out the stuff we have discussed in the info pannel and the botton left display
